velox/connectors/hive/HiveWriteProtocol.cpp

```diff
@@ -53,29 +57,46 @@ HiveNoCommitWriteProtocol::getWriterParameters(
       connectorQueryCtx->taskId(),
       connectorQueryCtx->driverId(),
       makeUuid());
+  auto targetDirectory = hiveWriteInfo->partitionDirectory().has_value()
```

Comment:
this code repeats twice; consider extracting a helper method

```
auto targetDirectory = makePartitionDirectory(hiveWriteInfo->partitionDirectory(), hiveTableWriteHandle->locationHandle()->targetPath());
```

Reply:
Got it. Removing the repetition.